packages/navigator/src/browser/navigator-filter.ts

```diff
@@ -60,14 +60,10 @@ export class FileNavigatorFilter {
         this.emitter.fire(undefined);
     }
 
-    protected onPreferenceChanged(event: PreferenceChangeEvent<FileNavigatorConfiguration>): void {
-        let hasChanged = false;
+    protected onFilesPreferenceChanged(event: PreferenceChangeEvent<FileSystemConfiguration>): void {
         const { preferenceName, newValue } = event;
-        if (preferenceName === 'navigator.exclude') {
+        if (preferenceName === FILES_EXCLUDE_PREFERENCE) {
```

Comment:
fyi: you don't need constants, it is statically typed here, try to use another property, ts won't let you

Reply:
@akosyakov Thanks! I see what you mean, but I use `files.exclude` in 3 places, that's why I've decided to use constant. For me it is better to change value in one place instead of doing it in 3 different places.
I've changed it a bit:
`const FILES_EXCLUDE_PREFERENCE: keyof FileSystemConfiguration = 'files.exclude';`